show-land/sass/_typography.scss

```diff
@@ -0,0 +1,8 @@
+body {
+    font-family: "Lato", sans-serif;
+    font-weight: 400;
+    /* font-size: 16px; */
+    line-height: 1.7;
+    color: $colorWihte;
+    }
+    
```

Comment:
You can write like this:

`body {`
`  font-family: "Lato", sans-serif;`
 ` font-weight: 400;`
 ` /* font-size: 16px; */`
 ` line-height: 1.7;`
 ` color: $colorWihte;`
`}`

Reply:
it's more cleaner

Reply:
Well, I wrote the same way